package.json

```diff
@@ -29,7 +29,7 @@
 		{
 			"gzip": false,
 			"running": false,
-			"limit": "6 KB",
+			"limit": "6.6 KB",
```

Comment:
this should never be changed without further approval from other from the team

Reply:
I've reverted the changes, but this will now fail the `size-limit` check since it goes over the limit.
I've tried to reduce the introduced changes to the minimum, I can not find a way to go any lower than this.

For the `modern/fronted` the size before was 5.9 KB, I can not make the changes match the available `0.1 KB` the poly-fill version is more generous in terms of space but the changes introduced add `0.56 KB` without a need for specific poly-fills.

I don't see a way to correct this behavior of Chrome without adding this much JS.

Let me know if you have any feedback on this. Thank you!